wellregistry/registry/tests/test_admin.py

```diff
@@ -7,10 +7,68 @@
 from django.http import HttpRequest
 from django.test import TestCase
 
-from ..admin import RegistryAdmin
+from ..admin import RegistryAdmin, RegistryAdminForm
 from ..models import AgencyLookup, Registry
 
 
+class TestRegistryFormAdmin(TestCase):
+
+    def setUp(self):
+        self.form_data = {
+            'display_flag': False,
+            'agency': 'USGS',
+            'site_no': '1234567',
+            'site_name': 'My test site',
```

Comment:
For each single field that needs to be non-blank, you will need to add it to the form_data.